pagopa/io-app
# fix: [IA-630] Profile birthdate is formatted considering the device locale

## Short description
The profile birthdate in the `fiscalcode fac-simile` screen is formatted considering the device locale instead of as UTC date.

## List of changes proposed in this pull request
- add new util function `formatDateAsShortFormatUtc` to handle this cases
- add tests

| before  | after |
| ------------- | ------------- |
![before](https://user-images.githubusercontent.com/822471/149515191-c93bd2b2-f1ec-4bb6-949d-3ad512e67d7c.png) | ![after](https://user-images.githubusercontent.com/822471/149515173-6ba0b166-36bf-4c81-820d-b12875a1df85.png)


## How to reproduce
- use the dev-server (**make sure to have the last master commit**)
- check the `fiscalcode fac-simile` birthdate should be `06/01/1990` (it's ok ✅ )
- [change your machine timezone ](https://www.macinstruct.com/tutorials/how-to-set-your-macs-time-zone/) to New York
- lock and unlock the emulator to make the time changes effective
- reload the app
- birthdate is `05/01/1990` 🐞
